PyInstaller/hooks/hook-lib2to3.py

```diff
@@ -0,0 +1,15 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+# This is needed to bundle draft3.json and draft4.json files that come
+# with jsonschema module
```

Comment:
Thanks for the pull-request. But this comment seams not to match what you wrote in the pul-reqeest's text.